test_schemas/en/test_routing_not_affected_by_answers_not_on_path.json

```diff
@@ -62,21 +62,19 @@
                             },
```

Comment:
This is in the out of scope on the card?. The change is fine, skip condition can be migrated another time.